module/Organizations/src/Organizations/Repository/Filter/ListJobQuery.php

```diff
@@ -34,12 +35,15 @@ class ListJobQuery extends AbstractPaginationQuery
      */
     public function createQuery($params, $queryBuilder)
     {
-        $id = $params['organization_id'];
+        $organization = $params['organization'];
         $queryBuilder
             ->field('organization')
-            ->equals($id)
+            ->equals($organization)
```

Comment:
@kilip 
This will produce an exception on the profile detail page: 
<code>Cannot create a DBRef for class Organizations\Entity\Organization without an identifier.</code>

However, using <code>$organization->getId()</code> works.

According to what my composer says, there is everything up to date. I wonder, why you do not get this error.